bread/bread.py

```diff
@@ -279,18 +279,16 @@ def get_queryset(self):
 
     def get_context_data(self, **kwargs):
         data = super(BrowseView, self).get_context_data(**kwargs)
-        o = self.request.GET.get('o', False)
-        if o:
-            data['o'] = o
-        q = self.request.GET.get('q', False)
-        if q:
-            data['q'] = q
+        data['o'] = self.request.GET.get('o', False)
+        data['q'] = self.request.GET.get('q', False)
```

Comment:
On Tue, May 19, 2015 at 2:00 PM, Vinod Kurup <notifications@github.com>
wrote:

> I think the empty string would be a better fallback, because otherwise
> Django converts False to the string 'False', so everyone's searchbox will
> start out with 'False' in it


​Yes, I discovered that myself and changed it to empty string.